packages/core/admin/server/src/services/__tests__/encryption.test.ts

```diff
@@ -0,0 +1,61 @@
+import crypto from 'crypto';
+import encryption from '../encryption';
+
+describe('Encryption Service', () => {
+  const ENCRYPTION_KEY = crypto.randomBytes(32).toString('hex');
+
+  beforeEach(() => {
+    global.strapi = {
+      config: {
+        get: jest.fn((key) => {
+          if (key === 'admin.secrets.encryptionKey') {
+            return ENCRYPTION_KEY;
+          }
+          return undefined;
+        }),
+      },
+      log: {
+        warn: jest.fn(),
+      },
+    } as any;
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe('encrypt', () => {
+    test('encrypts and returns a colon-separated string', () => {
```

Comment:
```suggestion
    test('encrypts and returns a three-part colon-separated string (iv, auth, enrypted)'), () => {
```
just because I was confused why we had this test 😆